client.go

```diff
@@ -29,6 +29,20 @@ type Client struct {
 	chCompany chan []*models.CompanyModel
 	flush    chan chan struct{}
 	interval time.Duration
+	responseETag string
+}
+
+func (c *Client) GetETag() string {
+	return c.responseETag
+}
+
+/*
+ * Get X-Moesif-Config-Etag from response headers
+ */
+func (c *Client) fetchETagFromHeader(headers http.Header) {	
+	if er, ok := headers["X-Moesif-Config-Etag"]; ok {
+		c.responseETag = er[0]
```

Comment:
nitpic: 'er' can be named as 'eTag' or something?. Also why is is it er[0]. header[''key'] returns value in that header right? Is it array?